models/pancakeswap/bnb/pancakeswap_v2_bnb_amm_trades.sql

```diff
@@ -21,22 +21,21 @@ WITH dexs AS
     SELECT
         t.evt_block_time                                                                AS block_time,
         t.to                                                                            AS taker,
-        ''                                                                              AS maker,
-        CASE WHEN amount0Out = '0' THEN amount1Out ELSE amount0Out END                  AS token_bought_amount_raw,
-        CASE WHEN amount0In = '0' OR amount1Out = '0' THEN amount1In ELSE amount0In END AS token_sold_amount_raw,
-        cast(NULL as double)                                                            AS amount_usd,
-        CASE WHEN amount0Out = '0' THEN f.token1 ELSE f.token0 END                      AS token_bought_address,
-        CASE WHEN amount0In = '0' OR amount1Out = '0' THEN f.token1 ELSE f.token0 END   AS token_sold_address,
+        CAST(NULL AS VARBINARY)                                                         AS maker,
+        CASE WHEN amount0Out = UINT256 '0' THEN amount1Out ELSE amount0Out END          AS token_bought_amount_raw,
+        CASE WHEN amount0In = UINT256 '0' OR amount1Out = UINT256 '0' THEN amount1In ELSE amount0In END AS token_sold_amount_raw,
+        NULL                                                            AS amount_usd,
+        CASE WHEN amount1Out = UINT256 '0' THEN f.token1 ELSE f.token0 END              AS token_bought_address,
```

Comment:
typo to change this from `amount0Out` to `amount1Out`?
looks like line 25 above is using `amount0Out` too

Reply:
also looks like ethereum version keep `amount0`

Reply:
Oh good catch. I had to fix this for Uniswap too. Must be a search and replace error